### PR TITLE
[env/burn-staging] design update for room modal

### PR DESCRIPTION
**Pull request by denisdimitrov**
Partially fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1053

- Redesign of room portals and the way they show content.

zxczxc

### CODE REVIEW

File: src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss [hunk 1]
```diff
@@ -2,31 +2,40 @@
 
 .ScheduleItem {
   display: flex;
-  padding: 10px 16px;
-  margin-top: 10px;
-  margin: 10px -16px 0px -16px;
+  padding: 10px 0;
```
Comment (dmitrymarchuk) [inline src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss:5]: Maybe extract constants over this file? But if it's from an old schedule and is eventually be removed then never mind.

**Review by dmitrymarchuk — CHANGES_REQUESTED** [review]

File: src/components/templates/PartyMap/components/RoomModal/RoomModal.scss [hunk 2]
```diff
@@ -3,39 +3,38 @@
 $border-radius: 22px;
 $spacing: 20px;
 
-.room-modal {
-  padding: $spacing;
-
-  background-color: $saturated-black;
+.RoomModal {
```
[the remark this answers is not in the document]
Comment (sunny-viktoryia) [reply src/components/templates/PartyMap/components/RoomModal/RoomModal.scss:6]: can you add a bit space between "by" and host?
image

File: src/components/templates/PartyMap/components/RoomModal/RoomModal.scss [hunk 3]
```diff
@@ -3,39 +3,38 @@
...
+  max-height: 80vh;
+  background-color: $dark;
   background-repeat: no-repeat;
   background-size: cover;
 
-  h2 {
-    text-align: center;
+  // This overrides the default black background color of react-boostrap's modal.
+  &__modal-content {
+    background-color: $dark !important;
```
Comment (sunny-viktoryia) [inline src/components/templates/PartyMap/components/RoomModal/RoomModal.scss:16]: can we get rid of the `!important` here?

File: src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss (hunk 1 above)
Comment (denisdimitrov) [reply src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss:5]: Yes, it's from the old schedule.

File: src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx [hunk 4]
```diff
@@ -59,10 +59,21 @@ export const RoomModal: React.FC<RoomModalProps> = ({
...
+    >
+      <Modal.Body className="RoomModal__modal-body">
+        <div className="RoomModal">
+          <RoomModalContent
+            room={room}
+            venueEvents={venueEvents}
+            venue={venue}
+          />
+        </div>
+      </Modal.Body>
```
[the remark this answers is not in the document]
Comment (denisdimitrov) [reply src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx]: Thank you, I didn't know about this.

Fixed in: https://github.com/sparkletown/sparkle/pull/2180/commits/c6c01ee3e7f376edf5ecc79762683c295c04703e

**Comment by denisdimitrov** [comment]
@dmitrymarchuk @sunny-viktoryia Ready for re-review.

**Review by dmitrymarchuk — APPROVED** [review]
Great!

**Pull request merged**